packages/web/src/components/comments/CommentHeader.tsx

```diff
@@ -19,11 +19,10 @@ type CommentHeaderProps = {
 
 export const CommentHeader = (props: CommentHeaderProps) => {
   const { isLoading } = props
-  const { handleMuteEntityNotifications, isEntityOwner, commentCount } =
-    useCurrentCommentSection()
+  const { isEntityOwner, commentCount } = useCurrentCommentSection()
   const { motion } = useTheme()
   const popupMenuItems: PopupMenuItem[] = [
-    { onClick: handleMuteEntityNotifications, text: messages.turnOffNotifs }
+    { onClick: () => {}, text: messages.turnOffNotifs }
```

Comment:
😁 
I'm assuming this is because the item is required to have an impl for this function but this isn't meant to be a clickable item?
Does it need a preventDefault or anything like that to keep it from bubbling up?

Reply:
This is just unimplemented at the moment. Its changing with a PR Dylan has open right now